EIPS/eip-hard-gas-target.md

```diff
@@ -0,0 +1,51 @@
+---
+eip: 3382
+title: Hardcoded Block Gas Limit
+author: Philippe Castonguay (@PhABC)
+discussions-to: xxxx
```

Comment:
Please add a discussion link when you're ready for this to be merged.

Reply:
Added